onnxscript/autocast.py

```diff
@@ -18,7 +18,11 @@ def cast_inputs(get_type_info, cast, opschema, *args):
     This is used by the converter in a static-mode, as well as by the eager-mode
     execution in a dynamic-mode.
     """
-    if opschema is not None:
+    # original code:
+    # if opschema is not None:
```

Comment:
Suggest removing line 21 and 22 since code history will record this and we don't need to add comments for it.